src/sentry/api/endpoints/project_details.py

```diff
@@ -229,6 +231,11 @@ def put(self, request, project):
 
         if changed:
```

Comment:
coverage doesn't seem happy about either of these branches, might be worthwhile?

Reply:
mm now it says 100% of diff hit, so i think it's ok?